### PR TITLE
Feature/form validation

### PR DESCRIPTION
**Pull request by Randyo28**
## Description

Created form validation for the new user registration form by using Ant D error handling functions. Previously there wasn't form validations and to check if fields were empty or doesn't have minimum amount of characters.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by fullstackcaveman — COMMENTED** [review]
I'm not sure of the package policies but maybe there should be mention of added/updated packages.

Looks good otherwise!

**Review by fullstackcaveman — APPROVED** [review]
Forgot to approve! ;)

File: src/components/pages/Login/SignupPage.js [hunk 1]
```diff
@@ -1,19 +1,13 @@
-import React, { useState } from 'react';
```
Comment (nicholasbus) [inline src/components/pages/Login/SignupPage.js:1]: removing unused vars 👍

**Review by nicholasbus — APPROVED** [review]
Form looks great :)

**Pull request merged**